packages/@tinacms/datalayer/src/node.ts

```diff
@@ -0,0 +1,141 @@
+import type { IncomingMessage, ServerResponse } from 'http'
```

Comment:
I am hesitant to revisit this again... but it feels like given that @tinacms/datalayer is currently just a facade for @tinacms/graphql this should probably go into @tinacms/graphql for now (in a backends directory). And actually this would probably be better in a separate self-hosting package, since it is only intended for that (whereas the other stuff in @tinacms/datalayer & @tinacms/graphql gets shared with tina cloud)

Reply:
It was my understanding that `@tinacms/datalayer` would be that package. Is it still used on TinaCloud?

Reply:
> It was my understanding that @tinacms/datalayer would be that package.

I don't recall us having any discussion about that, other than when were using @tinacms/datalayer to wrap up everything that was not open source (before we changed licensing)

It looks like only the older versions of @tinacms/graphql in tina cloud have the older @tinacms/datalayer dependency, so it it is probably ok to put the backend stuff here (though a bit confusing how it is morphing over time..). If we leave it there, I would move it to a sub folder called backend or api instead of leaving it at the root of the source tree.